required/examples/basics/video.js

```diff
@@ -1,26 +1,45 @@
-var renderer = PIXI.autoDetectRenderer(800, 600, { transparent: true });
-document.body.appendChild(renderer.view);
+var app = new PIXI.Application(800, 600, { transparent: true });
+document.body.appendChild(app.view);
 
-// create the root of the scene graph
-var stage = new PIXI.Container();
+// Create play button that can be used to trigger the video
+var button = new PIXI.Graphics()
+    .beginFill(0x0, 0.5)
+    .drawRoundedRect(0, 0, 100, 100, 10)
+    .endFill()
+    .beginFill(0xffffff)
+    .moveTo(36, 30)
+    .lineTo(36, 70)
+    .lineTo(70, 50);
 
-// create a video texture from a path
-var texture = PIXI.Texture.fromVideo('required/assets/testVideo.mp4');
+// Position the button
+button.x = (app.renderer.width - button.width) / 2;
+button.y = (app.renderer.height - button.height) / 2;
 
-// create a new Sprite using the video texture (yes it's that easy)
-var videoSprite = new PIXI.Sprite(texture);
+// Enable interactivity on the button
+button.interactive = true;
+button.buttonMode = true;
 
-videoSprite.width = renderer.width;
-videoSprite.height = renderer.height;
+// Add to the stage
+app.stage.addChild(button);
 
-stage.addChild(videoSprite);
+// Listen for a click/tap event to start playing the video
+// this is useful for some mobile platforms
```

Comment:
// Listen for a click/tap event to start playing the video
// this is useful for some mobile platforms. For example: 
// ios9 and under cannot render videos in PIXI without a polyfill - https://github.com/bfred-it/iphone-inline-video
// ios10 and above require a click/tap event to render videos that contain audio in PIXI. Videos with no audio track do not have this requirement